src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/PartySvcClient.java

```diff
@@ -16,9 +47,70 @@
    * @return the party object
    * @throws RestClientException something went wrong making http call
    */
-  PartyDTO requestParty(SampleUnitDTO.SampleUnitType sampleUnitType, String sampleUnitRef)
-      throws RestClientException;
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.